modules/customeradd.php

```diff
@@ -93,6 +93,11 @@
 	if($customeradd['name'] == '' && !$customeradd['type'])
 		$error['name'] = trans('First name cannot be empty!');
 	
+	if (ConfigHelper::getConfig('phpui.add_customer_group_required', false)) {
```

Comment:
jak to opakowywuje checkValue i w configu wspiszę cokolwiek to wali błędem, że w lini 117 configHelper jest nie prawidłowa wartość "cokolwiek". Jak zrezygnuję z opakowywanie to wprowadzenie czegokolwiek daje true